src/Components/SortableMovieList.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MovieList from './MovieList';
+import GenreList from './GenreList';
+
+const backGroundUrl = require("../images/fancy-pants.jpg")
+const upArrow = require('../images/upArrow.png');
+const downArrow = require('../images/downArrow.png');
+const styles = {
+    arrowImageStyle: {
+        padding: '0',
+        width: '36px',
+        backgroundColor: 'transparent',
+        verticalAlign: 'middle',
+        border: 'none',
+    },
+    buttonStyle: {
+        textTransform: 'uppercase',
+        height: '24px',
+        background: '#222244',
+        color: '#9999aa',
+        fontSize: '14px',
+        textAlign: 'center',
+        border: '1px #444455',
+        boxShadow: '0 0 3px #9999aa',
+        margin: '8px',
+        cursor: 'pointer'
+    },
+    listStyle: {
+        backgroundImage: `url(${backGroundUrl})`,
+        backgroundAttachment: 'fixed',
+        position: 'relative',
+        paddingTop: '40px',
+        display: 'flex',
+        width: '100%',
+        flexFlow: 'row wrap',
+        justifyContent: 'center' 
+    },
+    sortRow: {
+        width: '100%',
+        float: 'left'
+    }
+}
+
+class SortableMovieList extends Component {
+    constructor(props){
+        super(props);
```

Comment:
Thanks. I'll use this. I remember tying to change this but ended up having difficulties with passing the props correctly. I'm going to try to do a few very simple apps with no styling and play around to get a sense for how this works, better.